core/templates/pages/topic-viewer-page/stories-list/topic-viewer-stories-list.component.html

```diff
@@ -36,9 +30,23 @@ <h2 class="tab-title">{{ topicName }}</h2>
     padding-bottom: 2vh;
   }
 
+  .oppia-stories-list-container .oppia-page-card {
+    border-radius: 0.25em;
+    margin-top: 10em;
+    padding-bottom: 7.5vh;
+    width: 704px;
+  }
+
+  @media(max-width: 1024px) {
+    .oppia-stories-list-container .oppia-page-card {
```

Comment:
I think this same class is defined in story-summary-tile as well. Can your emove it from there as well?